src/main/java/com/snowflake/kafka/connector/internal/SnowflakeIngestionServiceV1.java

```diff
@@ -127,9 +128,12 @@ public void ingestFile(final String fileName) {
   @Override
   public void ingestFiles(final List<String> fileNames) {
     if (fileNames.isEmpty()) {
+      LOGGER.info("No files to ingest");
```

Comment:
will be easier to grep if we would have the same part of the message in the same logs
```suggestion
      LOGGER.info("ingest files: [Nothing to ingest]");
```